drivers/arm/i2c_master.h

```diff
@@ -51,6 +51,22 @@
 #    define I2C1_SDA 7
 #endif
 
+#if defined(STM32F4XX) || defined (STM32F3XX)
```

Comment:
Only f1 use gpiov1, <https://www.playembedded.org/blog/chibioshal-design-an-object-oriented-approach/#Achieving_abstraction_using_a_layered_architecture_design>, so can this be aligned as per previous suggestions to:
```c
#    if defined(STM32F1XX)
#        define USE_GPIOV1
#    endif
`

Reply:
updated to the USE_GPIOV1